common/cf-solana/src/client/impls.rs

```diff
@@ -168,7 +170,6 @@ impl ibc::ClientStateCommon for ClientState {
     }
 }
 
-
```

Comment:
I think you did `cargo fmt` with non-nightly and that introduced a bit of diff noise. ;)

Reply:
oh yes maybe